### ISSUE TITLE
Download counts

### ISSUE THREAD

**Issue by auburnsummer** [auburnsummer/rhythm-cafe#31]
This can and should be approximate and fuzzed

 - Current site doesn't have the technical capability to track download count
 - Can we use an analytics solution for this? beware gdpr
 - Low priority prob

**Comment by auburnsummer** [comment]
hmmmmm: use Amplitude https://www.amplitude.com/ to track download counts. it gives 10 million actions per month for the free plan which is the best I've seen. I also put in the sign up form that I'm an executive for the Rhythm Cafe company lol

 -  emit an Amplitude action each time the copy link OR download buttons are pressed for a link, but only once per device?
 -  maybe this brings us into gdpr territory, need to check

**Comment by auburnsummer** [comment]
Matomo does this: https://matomo.org/faq/general/how-is-the-visitor-config_id-processed/

- `config_id is a randomly-seeded, privacy-enabled, time-limited hash of a limited set of the visitor’s settings and attributes.`
- `the config_id is only valid for 24 hours maximum and is then rotated, meaning the same visitor will have a different config_id each day.`

**Comment by auburnsummer** [comment]
As long as:

 - the device id is synthetic (i.e. it's an arbitrary ID I generate)
 - no personal information is linked with the device id
 - i have a privacy policy !!!!

we're fine to do analytics without a cookie notice.
